hack/lib/common.sh

```diff
@@ -37,3 +40,29 @@ function is_installed {
 function install_service_monitor_crd {
   kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.35/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
 }
+
+# k8s v1.18.2 equivalent versions of envtest tools.
+ETCD_VERSION=v3.4.3
+KUBE_APISERVER_VERSION=$K8S_VERSION
+KUBECTL_VERSION=$K8S_VERSION
+# curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
+
+function setup_envs {
```

Comment:
May it would be better centralized in a new file as install.sh or setup.sh for we do not end up with a big shell very hard to maintain. Also, I think would be nice cover with comments for we know where/why and etc the code has been used.

Reply:
Eventually I will. Just playing around with these tests for now.